tools/pipelines/templates/include-test-real-service.yml

```diff
@@ -245,7 +245,10 @@ jobs:
       # Install test and logger package
       - task: Npm@1
         displayName: 'npm install'
-        retryCountOnTaskFailure: 1
+        # ADO feeds have latency on the order of minutes before packages are available downstream. See:
+        # https://learn.microsoft.com/en-us/azure/devops/artifacts/concepts/upstream-sources?view=azure-devops#upstream-sources-health-status
+        # This pipeline installs packages which were published very recently relative to its runtime, hence the rather high retry count here.
+        retryCountOnTaskFailure: 5
```

Comment:
can we bunch all the instances? not just this one? as if it doesn't work in this case, it won't work in the others either

Reply:
just saw your comment. i guess this covers both e2e and stress, not sure about benchmarks or how those work

Reply:
yeah, this covers benchmarks, dds fuzz, stress, and the e2e service clients pipelines. The perf tests don't seem to suffer from this as bad, I think because their npmrc points to the directly published feed. We can revisit if it becomes an issue.